src/leiningen/v.clj

```diff
@@ -103,15 +103,34 @@
   (let [v (str (or (version (:v project)) "UNKNOWN"))
         vk (str (or (:manifest-version-name (:v project)) "Implementation-Version"))]
     (-> project
-       (assoc-in [:version] v)
-       (assoc-in [:manifest vk] v))))
+        (assoc-in [:version] v)
+        (assoc-in [:manifest vk] v))))
+
+(defn- update-dependency [v d]
+  (if-not (second d)
+    (assoc d 1 v)
+    d))
+
+; => (update-dependency "42" ["abc" "1.0"])
```

Comment:
For consistency sake, please keep examples in README.  I'm looking forward to clojure.spec for some documentation capabilities, but until then I don't want to have to worry about keeping comments updated as code evolves.

Reply:
Will move that to the tests...